skins/wmde19/src/components/layout/Header.vue

```diff
@@ -1,54 +1,48 @@
 <template>
-	<div class="navbar is-fixed-top has-background-bright has-shadow">
-		<div class="container">
-			<div class="navbar-brand">
-				<a class="navbar-item" href="/">
-					<img :src="imagePath + '/images/logo-horizontal-wikimedia.svg'" alt="Wikimedia Deutschland" width="144" height="29">
-				</a>
-				<a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
-					<span aria-hidden="true"></span>
-					<span aria-hidden="true"></span>
-					<span aria-hidden="true"></span>
-				</a>
-			</div>
+    <div class="navbar is-fixed-top has-background-bright has-shadow">
+        <div class="container">
+            <div class="navbar-brand">
+                <a class="navbar-item" href="/">
+                    <img :src="imagePath + '/images/logo-horizontal-wikimedia.svg'" alt="Wikimedia Deutschland"
+                         width="144" height="29">
+                </a>
+                <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false"
+                   v-bind:class="{ 'is-active': showNavbarBurger }" v-on:click="showNavbarBurger = !showNavbarBurger">
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+                </a>
+            </div>
 
-			<div class="navbar-menu" id="navMenu">
-				<div class="navbar-start">
-					<a class="navbar-item">
-						Spendenkommentare
-					</a>
-					<a class="navbar-item">
-						Häufige Fragen
-					</a>
-					<a class="navbar-item">
-						Mittelverwendung
-					</a>
-					<a class="navbar-item">
-						Spenden
-					</a>
-				</div>
-			</div>
-		</div>
-	</div>
+            <div class="navbar-menu" id="navMenu"
+                 v-bind:class="{ 'is-active': showNavbarBurger }" v-on:click="showNavbarBurger = !showNavbarBurger">
+                <div class="navbar-start">
+                    <a class="navbar-item">
+                        Spendenkommentare
+                    </a>
+                    <a class="navbar-item">
+                        Häufige Fragen
+                    </a>
+                    <a class="navbar-item">
+                        Mittelverwendung
+                    </a>
+                    <a class="navbar-item">
+                        Spenden
+                    </a>
+                </div>
+            </div>
+        </div>
+    </div>
 </template>
 
 <script>
-export default {
-	name: 'Header',
-	props: [ 'imagePath' ],
-};
-
-// Navbar burger animation
-document.addEventListener( 'DOMContentLoaded', () => {
-	const $navbarBurgers = Array.prototype.slice.call( document.querySelectorAll( '.navbar-burger' ), 0);
-	if ( $navbarBurgers.length > 0 ) {
-		$navbarBurgers.forEach( el => {
-			el.addEventListener( 'click', () => {
-				el.classList.toggle( 'is-active' );
-				document.querySelector('.navbar-menu').classList.toggle('is-active');
-			} );
-		} );
-	}
-} );
-
+	export default {
```

Comment:
If you're feeling adventurous, have a peek at other components and do a `<script type="ts">` and see what happens and how you need to modify this component to become a TypeScript component.